rest_framework_tracking/management/commands/clearapilogs.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand, CommandError
+from rest_framework_tracking.models import APIRequestLog
+from optparse import make_option
+import datetime
+from django.utils import timezone
+
```

Comment:
Why not try to write a unit test which creates some test log items and to test and validate your delete code?

Reply:
I just fixed the previous issue and I tested it with the `days_num` argument and without and everything works as expected.